modules/backup/main.tf

```diff
@@ -99,6 +99,7 @@ resource "google_workflows_workflow" "sql_backup" {
     backupRetentionTime      = var.backup_retention_time
     backupRunsListMaxResults = var.backup_runs_list_max_results
   })
+  deletion_protection = var.deletion_protection
```

Comment:
Lets bump [min version](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/8cb3a03acaffef72e8a924c6ff0d11a0f645c1ef/modules/backup/versions.tf#L22) to `6.11.0` when this was introduced

https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md#6110-november-11-2024

Reply:
Done!